src/routes/email/email.controller.delete-email.spec.ts

```diff
@@ -97,6 +112,9 @@ describe('Email controller delete email tests', () => {
       })
       .expect(204)
       .expect({});
+
+    expect(emailApi.deleteEmailAddress).toBeCalledTimes(1);
```

Comment:
Nitpick: according to Jest docs, `toBeCalledTimes` is deprecated in favor of `toHaveBeenCalledTimes` 🙂

Reply:
Good catch! [034c228](https://github.com/safe-global/safe-client-gateway/pull/994/commits/034c2286ce2058625a214fc4b44e06d40f3be97b)